daffodil-runtime1/src/main/scala/org/apache/daffodil/layers/LineFoldedTransformer.scala

```diff
@@ -296,10 +296,6 @@ class LineFoldedInputStream(mode: LineFoldMode, jis: InputStream)
         case GotCRLF => {
           c = jis.read()
           c match {
-            case -1 => {
-              state = Buf2 // buffering up the LF
-              return '\r'
-            }
```

Comment:
Although the code here is the same as the default case at the end, the comment is different. Can you adjust the comment of the default case so it mentions what this case is checking for?, something like "CRLF followed by other not sp/tab, or end of data. Buffer up to the LF"